mackup/config.py

```diff
@@ -206,7 +207,11 @@ def _parse_path(self):
                 raise ConfigError("The required 'path' can't be found while"
                                   " the 'file_system' engine is used.")
 
-        return str(path)
+        # Python 2 and python 3 byte strings are different.
+        if sys.version_info[0] < 3:
+            return str(path)
+        else:
+            return path.decode("utf-8")
```

Comment:
ok thats fine.
Sorry to be a bother, but can you keep 1 return at the end of the function? Assign a temp var, thats fine.